### PR TITLE
Added health checks with flask-healthz

### PR DESCRIPTION
**Pull request by rod7760**
Resolves: #352

Signed-off-by: Riccardi Dalexis <rickdalexisdev@gmail.com>

### CODE REVIEW

**Comment by rod7760** [comment]
I resolved the failing lint checks. My apologies.

File: mirrormanager2/health_checks.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+
+def liveness():
+    pass
+
+
+def readiness():
+    try:
+        DB.manager.get_status()
+    except OperationalError as err:
```
Comment (abompard) [inline mirrormanager2/health_checks.py:14]: This looks good, but we can do even better! The sqlalchemy-helpers doc has an example implementation of a health check: https://sqlalchemy-helpers.readthedocs.io/en/latest/user.html#openshift-health-checks

**Review by abompard — CHANGES_REQUESTED** [review]
Thanks for your contribution! There's a possible improvement that I noted, but it's overall very good!

File: mirrormanager2/health_checks.py (hunk 1 above)
Comment (rod7760) [reply mirrormanager2/health_checks.py]: Ah. I'm unsure how I missed that, but PR has been updated. I also realized I could simplify the tests.

Thanks

**Review by abompard — APPROVED** [review]
Looks great, thanks again!

**Pull request merged**